includes/menu.inc

```diff
@@ -17,41 +18,41 @@ function oe_bootstrap_theme_preprocess_menu(&$variables, $hook) {
     return;
   }
 
-  $items = $variables['items'];
-  foreach ($items as $key => $item) {
-    $class = $item['in_active_trail'] ? ['active'] : [];
-    $menu_item = [
+  $variables['pattern_items'] = [];
+  foreach ($variables['items'] as $key => $item) {
+    $pattern_item = [
       'label' => $item['title'],
-      'path' => $item['url']->toString(),
-      'attributes' => new Attribute(['class' => $class]),
+      'path' => $item['url'] instanceof Url ? $item['url']->toString() : '',
```

Comment:
looking into it, it's always an Url object, even if we use `<nolink>` route, the result of toString of that is indeed an empty string, BUT since the field is core required, its always gonna be a Url object, this wont change.
So we don't need this default value
```
'path' => $item['url']->toString();
```